src/engine/identification.md

```diff
@@ -7,15 +7,17 @@ Engine API structures and methods specified for client version specification.
 <!-- START doctoc generated TOC please keep comment here to allow auto update -->
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
 
-- [Client Version Specification](#client-version-specification)
-- [Structures](#structures)
-  - [ClientCode](#clientcode)
-  - [ClientVersionV1](#clientversionv1)
-- [Methods](#methods)
-  - [engine_getClientVersionV1](#engine_getclientversionv1)
-    - [Request](#request)
-    - [Response](#response)
-    - [Specification](#specification)
+- [Engine API -- Client Version Specification](#engine-api----client-version-specification)
```

Comment:
Could you please revert the ToC changes, just for the sake of making the change cleaner

Reply:
Fixed sorry about that :pray: